client/extensions/woocommerce/state/sites/payment-methods/actions.js

```diff
@@ -49,3 +52,45 @@ export const fetchPaymentMethods = ( siteId ) => ( dispatch, getState ) => {
 			dispatch( setError( siteId, getAction, err ) );
 		} );
 };
+
+export const paymentMethodSaveSuccess = ( siteId, data ) => {
```

Comment:
Does this need to be a separate function and/or exported? (reading up, I see that `fetchPaymentMethodsSuccess` also exists, but I have the same issue there — I don't see why it should be a separate function, it can make scanning the action code harder)